mailu/templates/front/ingress.yaml

```diff
@@ -18,6 +18,9 @@ metadata:
     {{- if .Values.commonAnnotations }}
     {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
     {{- end }}
+    {{- if and (eq .Values.ingress.type "traefik") .Values.ingress.enabled .Values.ingress.traefik.insecureSkipVerify }}
```

Comment:
This is needed instead on the `mailu-front` service.

Also, the `https/443` port on the `mailu-front` deployment should be enabled again, otherwise traefik refuses to link the ingress as the pod does not expose `https`